eng/common/scripts/Helpers/DevOps-WorkItem-Helpers.ps1

```diff
@@ -1015,3 +1015,64 @@ function UpdateValidationStatus($pkgvalidationDetails, $BuildDefinition, $Pipeli
     Write-Host "[$($workItem.id)]$LanguageDisplayName - $pkgName($versionMajorMinor) - Updated"
     return $true
 }
+
+function Get-ReleasePlansForCPEXAttestation()
+{
+  $fields = @()
+  $fields += "Custom.ProductServiceTreeID"
+  $fields += "Custom.ReleasePlanType"
+  $fields += "Custom.ProductType"
+  $fields += "Custom.DataScope"
+  $fields += "Custom.MgmtScope"
+
+  $fieldList = ($fields | ForEach-Object { "[$_]"}) -join ", "
+  $query = "SELECT ${fieldList} FROM WorkItems WHERE [System.WorkItemType] = 'Release Plan' AND [System.State] = 'Finished'"
+  $query += " AND [Custom.AttestationStatus] IN ('', 'Pending')"
+  $query += " AND [System.Tags] NOT CONTAINS 'Release Planner App Test'"
+  $query += " AND [System.Tags] NOT CONTAINS 'Release Planner Test App'"
+  $query += " AND [System.ChangedDate] >= @StartOfDay('-1')" # Filter for items changed yesterday only
+  $query += " AND [System.ChangedDate] < @StartOfDay('0')"
+
+  $workItems = Invoke-Query $fields $query
+  return $workItems
+}
+
+function Get-TriagesForCPEXAttestation()
+{
+  $fields = @()
+  $fields += "Custom.ProductServiceTreeID"
+  $fields += "Custom.ProductType"
+  $fields += "Custom.DataScope"
+  $fields += "Custom.MgmtScope"
+
+  $fieldList = ($fields | ForEach-Object { "[$_]"}) -join ", "
+  $query = "SELECT ${fieldList} FROM WorkItems WHERE [System.WorkItemType] = 'Triage' AND [System.State] IN ('Completed', 'New')"
```

Comment:
another valid value is `Triage updated`. This happens when a team recently updated the data.
We wanted to flag it as "hey something changed". so we should also consider that one
example: https://dev.azure.com/azure-sdk/Release/_queries/edit/18565